chall01/rpehkone.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+import sys
+
+if len(sys.argv) != 2 or not sys.argv[1]:
+	print("usage: " + sys.argv[0] + " <a-zA-Z string>")
+	exit(0)
+for i in sys.argv[1]:
+	if not (i.isalpha() and ord(i) < 128) and i != ' ':
+		print("usage: " + sys.argv[0] + " <a-zA-Z string>")
+		exit(0)
```

Comment:
well done using sys.argv[0] for the program name and checking string beforehand for errors
however, overall this check can be done in a more "pythonic" way: 
`if not sys.argv[1].replace(" ", "").isalpha():`